### PR TITLE
local: Run Airflow on docker-compose + locally

### PR DESCRIPTION
**Pull request by Neodia**
* Added docker-compose config

* Eased the process of running Airflow locally.
  This was needed because performance of
  Airflow within docker is not great.

closes issues-scoap3#1

### CODE REVIEW

File: data/article1_springer.xml [hunk 1]
```diff
@@ -0,0 +1,188 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (drjova) [inline data/article1_springer.xml:1]: why do we need these files int he PR?

File: README.md [hunk 2]
```diff
@@ -9,15 +9,17 @@ For it docker-compose has to be installed.After just run in it the command bello
...
 
 We need to install airflow.
 
 1. First, we should create virtual environment with pyenv:
 
 ```
-    pyenv global 3.7.10
-    export PYTHON_VERSION=3.7.10
+    export PYTHON_VERSION=3.8.13
+    pyenv global 3.8.13
```
Comment (drjova) [inline README.md]:
```suggestion
    pyenv install 3.8.13
    pyenv global 3.8.13
```

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+	airflow scheduler -D
+	echo -e "\033[0;32m Airflow Started. \033[0m"
+
+stop:
+	docker-compose down
+	cat $(WEBSERVER_PID) | xargs kill
+	cat $(TRIGGERER_PID) | xargs kill
+	cat $(SCHEDULER_PID) | xargs kill
+	rm *.out *.err
+	echo -e "\033[0;32m Airflow Stoped. \033[0m"
```
Comment (drjova) [inline Makefile]: you could use this editor config https://github.com/inspirehep/inspirehep/blob/master/.editorconfig (you can also committed in the repo)

File: README.md (hunk 2 above)
Comment (Neodia) [reply README.md]: Fixed

File: Makefile (hunk 3 above)
Comment (Neodia) [reply Makefile]: Added

File: data/article1_springer.xml (hunk 1 above)
Comment (Neodia) [reply data/article1_springer.xml:1]: They're part of the volume given for the SFTP server. I understand the argument if you think this is out of scope, should I remove this for this MR and put it when I'll start implementing the parser for Springer ?

**Review by drjova — APPROVED** [review]

**Pull request merged**